admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -240,6 +316,7 @@ private void setStatusRelatedFields(ReplicationField field, ReplicatorConfig con
     address.port(url.getPort());
     field.address(address);
     field.rootContext(StringUtils.isEmpty(url.getPath()) ? DEFAULT_CONTEXT : url.getPath());
+    field.isDisableLocal(site.isRemoteManaged());
```

Comment:
❓ Wouldn't have been consistent in naming be better?

Reply:
Whoops, yeah. Forgot to rename this method on the field.